tests/console/yast2_kdump_use_fadump.pm

```diff
@@ -29,7 +29,7 @@ sub run {
     save_screenshot;
     $fadump->get_navigation->ok();
     save_screenshot;
-    $restartinfo->confirm_reboot_needed();
+    $restartinfo->wait_restart_info_popup->get_restart_info_page->confirm_reboot_needed();
```

Comment:
Those are two calls:
```
$restartinfo->wait_restart_info_popup();
$restartinfo->confirm_reboot_needed();
```
If you chain them, you don't need the `get_restart_info_page` in the middle because you already have the $self, but `wait_restart_info_popup` should not return $self, because its purpose is just to 'wait'.

Reply:
Updated code and VR. Thanks.